File: WebClient/src/app/model/command.ts
export class ICommand {
    constructor(
        public Id:number,
        public CommandType:string,
        public Params:string,
        public Issued:Date,
        public Actioned:Date, 
        public DeviceId: number,              
        public CreatedAt:Date,
        public UpdatedAt:Date
    ){}
}

File: Device/DeviceController.Data/Command.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DeviceController.Data
{
    public class Command
    {
        public int Id;
        public int CommandType;
        public string Params;
        public int DeviceId;
        public DateTime? Issued;
        public DateTime? Actioned;
        public DateTime? CreatedAt;
        public DateTime? UpdatedAt;

    }
}
